lib/trino-parquet/src/main/java/io/trino/parquet/reader/StructColumnReader.java

```diff
@@ -38,9 +38,6 @@ public static BooleanList calculateStructOffsets(
         int maxRepetitionLevel = field.getRepetitionLevel();
         BooleanList structIsNull = new BooleanArrayList();
         boolean required = field.isRequired();
-        if (fieldDefinitionLevels == null) {
```

Comment:
`requireNotNull` just in case?

Reply:
Since we are dereferencing `fieldDefinitionLevels` in next line, requireNonNull is not necessary, we'll get a readable nullptr exception anyway (improvement in new java).